plugin/datasource/src/main/java/com/alibaba/nacos/plugin/datasource/mapper/AbstractMapper.java

```diff
@@ -42,28 +44,22 @@ public String select(List<String> columns, List<String> where) {
         sql.append("FROM ");
         sql.append(getTableName());
         sql.append(" ");
-        
-        if (where.size() == 0) {
+
+        if (CollectionUtils.isEmpty(where)) {
             return sql.toString();
         }
-        
-        sql.append("WHERE ");
-        for (int i = 0; i < where.size(); i++) {
-            sql.append(where.get(i)).append(" = ").append("?");
-            if (i != where.size() - 1) {
-                sql.append(" AND ");
-            }
-        }
+
+        appendWhereClause(where, sql);
         return sql.toString();
     }
-    
+
```

Comment:
Please don't change the indent.

Use nacos code style to reformat codes.

Reply:
I've fixed check style.